feature/experimental/p4rt/otg_tests/google_discovery_protocol_packetin_test/google_discovery_protocol_packetin_test.go

```diff
@@ -257,14 +279,19 @@ func TestMain(m *testing.M) {
 }
 
 // configInterfaceDUT configures the interface with the Addrs.
-func configInterfaceDUT(i *oc.Interface, a *attrs.Attributes, dut *ondatra.DUTDevice) *oc.Interface {
+func configInterfaceDUT(i *oc.Interface, a *attrs.Attributes, dut *ondatra.DUTDevice, hasVlan bool) *oc.Interface {
 	i.Description = ygot.String(a.Desc)
 	i.Type = oc.IETFInterfaces_InterfaceType_ethernetCsmacd
 	if deviations.InterfaceEnabled(dut) {
 		i.Enabled = ygot.Bool(true)
 	}
-
-	s := i.GetOrCreateSubinterface(0)
+	var s *oc.Interface_Subinterface
+	if hasVlan && deviations.P4RTGdpRequiresDot1QSubinterface(dut) {
```

Comment:
Please add 2xsubinterfaces to cover for production scenario. One subinterface to accept untagged frames and another subinterface to cover for GDP packets w/ dot1Q encapsulation. These subinterfaces should be applied only when a NOS satisfies deviation.